modules/300-prometheus/oss.yaml

```diff
@@ -16,4 +16,18 @@
   logo: https://raw.githubusercontent.com/trickstercache/trickster/master/docs/images/logos/trickster-logo.svg
   licence: Apache License 2.0
 
+- name: Mimir
+  link: https://github.com/trickstercache/trickster
+  description: A HTTP reverse proxy/cache for http applications and a dashboard query accelerator for time series databases.
+  logo: https://grafana.com/static/img/logos/logo-mimir.svg
+  licence: AGPL-3.0 license
+
+- name: Promxy
+  link: https://github.com/jacksontj/promxy
+  description: An aggregating proxy to enable HA prometheus
```

Comment:
The description does not describe what promxy actually does. There is a better version on its website.

> Promxy is a prometheus proxy that makes many shards of prometheus appear as a single API endpoint to the user. This significantly simplifies operations and use of prometheus at scale (when you have more than one prometheus host).

Reply:
changed